tests/sites_framework/tests.py

```diff
@@ -73,3 +75,28 @@ class ConfusedArticle(models.Model):
             )
         ]
         self.assertEqual(errors, expected)
+
+
+class ContextProcessorTest(TestCase):
+    def setUp(self):
+        self.factory = RequestFactory()
+
+    def test_context_processor(self):
+        site_obj = Site.objects.get_current()
+        request = self.factory.get("/")
+        context = site(request)
+        self.assertIn("site", context)
+        self.assertEqual(context["site"].name, site_obj.name)
+        self.assertEqual(context["site"].domain, site_obj.domain)
+
+    def test_context_processor_without_sites_framework(self):
+        server_url = "testserver"
+        apps = list(settings.INSTALLED_APPS)
+        apps.remove("django.contrib.sites")
+
+        with self.settings(INSTALLED_APPS=apps):
+            request = self.factory.get("/")
+            context = site(request)
+
+            self.assertEqual(context["site"].name, server_url)
+            self.assertEqual(context["site"].domain, server_url)
```

Comment:
I don't think this extra test provides any value - it's only verifying the behaviour of `get_current_site()`, which should be tested without the framework elsewhere.
```suggestion
```